packages/stencil/components/select/src/components/osds-select/osds-select.tsx

```diff
@@ -270,32 +292,41 @@ export class OsdsSelect implements OdsSelect<OdsStencilMethods<OdsSelectMethods>
 
     return (
       <Host {...{
-        class: `${disabled ? 'disabled' : ''}${opened ? ' opened' : ''}${this.hasError() ? ' ods-error' : ''}`,
+        class: `${disabled ? 'disabled' : ''}${isSurfaceOpen ? ' opened' : ''}${this.hasError() ? ' ods-error' : ''}`,
```

Comment:
isSurfaceOpen, is it working? Could not find it in DOM for class. Besides, maybe it won't be necessary since ocdk-surface (its class) already manages this. We can discuss about it